assets/artworks/empty

```diff
@@ -0,0 +1 @@
+
```

Comment:
this file?

Reply:
I'm used to make an empty file when I want to keep an empty folder on git. Usually when the user needs to manually put something in before the first start. The `images` folder isn't commited because the script will create it anyway.

Reply:
I see. I usually do a `mkdir -p` (or equivalent) when writing out